task/buildah/0.2/buildah.yaml

```diff
@@ -107,6 +107,10 @@ spec:
     description: Additional key=value labels that should be applied to the image
     type: array
     default: []
+  - default: "false"
+    description: This option is only available for remote tasks, it will add --privileged to podman run and --security-opt=label=disable --cap-add=all --device=/dev/fuse to buildah build
```

Comment:
I'll pretend I'm a Konflux user

What is a "remote" task? If this task is not one of those, why does it have this option?

Reply:
How could I remove this option in buildah task but leave it in buildah-remote tasks, after running ./hack/generate-buildah-remote.sh, this option will be removed from buildah-remote task either.

Reply:
Ah, it seems there is a recipe.yaml in each tasks might be used for that, let me try.

Reply:
Okay, I should append the remote param in task-generator/remote/main.go, updated.